jsreport-linux-appservice-docker/azuredeploy.json

```diff
@@ -0,0 +1,104 @@
+        "Standard_ZRS",
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Premium_LRS"
+      ],
+      "metadata": {
+        "description": "Describes the Storage Account Type"
+      }
+    }
+  },
+  "variables": {
+    "servicePlanName": "[concat('serviceplan',uniqueString(resourceGroup().id))]",
+    "StorageName": "[concat('jsreports', uniqueString(resourceGroup().id))]",
+    "storageAccountApiVersion": "2015-06-15",
+    "storageId": "[resourceId('Microsoft.Storage/storageAccounts', variables('StorageName'))]",
+    "storageConnectionStringPrefix":
+      "[concat('DefaultEndpointsProtocol=https;AccountName=',variables('StorageName'),';EndpointSuffix=core.windows.net',';AccountKey=')]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Web/serverfarms",
+      "name": "[variables('servicePlanName')]",
+      "apiVersion": "2016-03-01",
+      "location": "[resourceGroup().location]",
+      "sku": {
+        "name": "S1",
+        "capacity": 1
+      },
+      "kind": "docker",
+      "properties": {
+        "workerSizeId": 0,
+        "reserved": true,
+        "hostingEnvironment": ""
+      }
+    },
+    {
+      "type": "Microsoft.Web/sites",
+      "name": "[parameters('appName')]",
+      "apiVersion": "2016-03-01",
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "name": "[parameters('appName')]",
+        "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('servicePlanName'))]"
+      },
+      "dependsOn": [
+        "[resourceId('Microsoft.Web/serverfarms', variables('servicePlanName'))]"
+      ],
+      "resources": [
+        {
+          "name": "appsettings",
+          "type": "config",
+          "apiVersion": "2016-03-01",
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/sites', parameters('appName'))]"
+          ],
+          "tags": {
+            "displayName": "appSettings"
+          },
+          "properties": {
+            "DOCKER_CUSTOM_IMAGE_NAME": "jsreport/jsreport:2.1.0-full",
+            "extensions_authentication_admin_password": "password",
+            "extensions_authentication_admin_username": "admin",
+            "extensions_authentication_cookieSession_secret": "dasd321as56d1sd5s61vdv32",
+            "extensions_fsStore_persistence_provider": "azure-storage",
+            "extensions_fsStoreAzureStoragePersistence_accountName": "[variables('StorageName')]",
+            "extensions_fsStoreAzureStoragePersistence_accountKey": "[listkeys(variables('storageId'), variables('storageAccountApiVersion')).key1]",
+            "extensions_fsStoreAzureSbSync_connectionString": "[concat(variables('storageConnectionStringPrefix'), listkeys(variables('storageId'), variables('storageAccountApiVersion')).key1)]"
+          }
+        }
+      ]
+    },
+    {
+      "name": "[variables('StorageName')]",
+      "type": "Microsoft.Storage/storageAccounts",
+      "location": "[resourceGroup().location]",
```

Comment:
Add location parameter - see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#parameters

Reply:
Applied Changes to "location": "[parameters('location')]",